src/Apps/Settings/Routes/EditProfile/SettingsEditProfileRoute.tsx

```diff
@@ -13,6 +22,17 @@ const SettingsEditProfileRoute: React.FC<SettingsEditProfileRouteProps> = ({
 }) => {
   return (
     <GridColumns>
+      <Column span={8}>
```

Comment:
```tsx
<Column span={8} bg="black5" p={2} display="flex" gap={1}>
        <InfoIcon flexShrink={0} />

        <Text variant="sm-display">
          The information you provide here will be shared when you contact a
          gallery or make an offer.
        </Text>
      </Column>
```

However! Wondering if this should actually be a [banner](https://palette-storybook.artsy.net/?path=/story/components-banner--default)?

Reply:
My understanding is that Banner components are typically used for status updates or confirmations (confirmed in the code). I'm not sure if we should use it here. Is this a design question, or can we proceed with the existing approach? Open to suggestions!

Reply:
Yeah, that was also my question. I think typically we use the [Message](https://palette-storybook.artsy.net/?path=/story/components-message--default) for this sort of thing, though I do see the need for an icon slot.

Reply:
Anyway, fine with existing approach (with the CSS edits I propose) for the time being.

Reply:
sounds good!